xbmc/video/VideoInfoTag.cpp

```diff
@@ -591,6 +598,11 @@ void CVideoInfoTag::ParseNative(const TiXmlElement* movie, bool prioritise)
   {
     m_fRating = m_fRating / max_value * 10; // Normalise the Movie Rating to between 1 and 10
   }
+  const TiXmlElement* urElement = movie->FirstChildElement("userrating");
+  if (urElement && (urElement->QueryIntAttribute("max", &max_value) == TIXML_SUCCESS) && max_value >= 1)
+  {
```

Comment:
No need for the curly braces (yes I know they are also used above but we don't do that anymore).